src/View/mainpage/DicomView.py

```diff
@@ -53,6 +68,37 @@ def init_view(self):
         background_brush = QtGui.QBrush(
             QtGui.QColor(0, 0, 0), QtCore.Qt.SolidPattern)
         self.view.setBackgroundBrush(background_brush)
+    
+    def wheelEvent(self, event: QtGui.QWheelEvent):
+        modifiers = event.modifiers()
+        if modifiers == QtCore.Qt.ShiftModifier:
+            super().wheelEvent(event)
+        elif modifiers == QtCore.Qt.AltModifier:
+            super().wheelEvent(event)
+        elif modifiers == QtCore.Qt.ControlModifier:
+            angle = event.angleDelta() / 8
+            delta = angle.y() / 15
+            if delta > 0:
+                self.zoom_in()
+            else:
+                self.zoom_out()
+        else:
+            num_degrees = event.angleDelta() / 8
```

Comment:
make magic numbers in to constants, and make the divisors here variables (with the initial value of the variable being the appropriate constant).  Use names that will help explain what they are for (or just put in the URL from my previous comment).

Reply:
Yes Stuart, I will modify the code